jetty-util/src/main/java/org/eclipse/jetty/util/thread/ShutdownThread.java

```diff
@@ -48,6 +48,7 @@
      */
     private ShutdownThread()
     {
+        super("ShutdownThread");
```

Comment:
We should add some "Jetty" naming here, because shutdown threads may be registered by other components (e.g. Spring) and have the same name because they are too "ShutdownThread" -- it's a too common name.